domains/preston-code.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "preston-riley",
+    "email": "pres1234569@gmail.com"
+  }
+}
+{
+  "record": {
+    "CNAME": "preston-code.surge.sh"
+  }
+}
```

Comment:
invalid file, it should be like this:
`{
	"owner": {
	  "username": "preston-riley",
	  "email": "pres1234569@gmail.com"
	},
	"record": {
	  "CNAME": "preston-code.surge.sh"
	}
  }`

Reply:
done